scripts/poo-tracker.coffee

```diff
@@ -0,0 +1,32 @@
+Redis = require 'redis'
+Url = require 'url'
+
+POO_TRACKER_KEY = "poops"
+POO_LATEST_KEY = "poops:latest_message"
+
+info = Url.parse process.env.POO_REDIS_URL or "redis://localhost:6379/0"
+redis_client = Redis.createClient(info.port, info.hostname)
+redis_client.auth info.auth.split(":")[1] if info.auth
+
+module.exports = (robot) ->
+  checkRedisForShit = ->
+    redis_client.lindex POO_TRACKER_KEY, -1, (err, reply) ->
+      return console.error("Failed lindex with key '#{POO_TRACKER_KEY}' and index -1: #{err}") if err
+      return if reply == null
+
+      room = process.env.HUBOT_IRC_ROOMS or "#arrakis"
+      robot.messageRoom room, "#{reply}"
```

Comment:
Changed this to robot.say, which according to `hubot-irc` takes a string and not a hash. So maybe give his another shot?